crates/oxc_prettier/src/format/mod.rs

```diff
@@ -1013,7 +1021,7 @@ impl<'a> Format<'a> for ImportAttribute {
 impl<'a> Format<'a> for ExportNamedDeclaration<'a> {
     fn format(&self, p: &mut Prettier<'a>) -> Doc<'a> {
         let mut parts = p.vec();
-        parts.push(module::print_module_specifiers(p, &self.specifiers, false));
+        parts.push(module::print_module_specifiers(p, &self.specifiers, false, false));
```

Comment:
When there are multiple boolean arguments like this, it is sometimes good to document it like this: `, /* is_default */ false, /* is_namespace */ true)` for readability.